chapters/data-analysis.tex

```diff
@@ -1,292 +1,487 @@
+The folder must be encrypted if it is shared in an insecure fashion,\sidenote{\url{https://dimewiki.worldbank.org/wiki/Encryption}}
 and it must be backed up in a secure offsite location.
 Everything else can be replaced, but raw data cannot.
 Therefore, raw data should never be interacted with directly.
 
-Instead, the first step upon receipt of data is \textbf{de-identification}.\sidenote{\url{https://dimewiki.worldbank.org/wiki/De-identification}}
-\index{de-identification}
-There will be a code folder and a corresponding data folder
-so that it is clear how de-identification is done and where it lives.
-Typically, this process only involves stripping fields from raw data,
-naming, formatting, and optimizing the file for storage,
-and placing it as a \texttt{.dta} or other data-format file
-into the de-identified data folder.
-The underlying data structure is unchanged:
-it should contain only fields that were collected in the field,
-without any modifications to the responses collected there.
-This creates a survey-based version of the file that is able
-to be shared among the team without fear of data corruption or exposure.
-Only a core set of team members will have access to the underlying
-raw data necessary to re-generate these datasets,
-since the encrypted raw data will be password-protected.
-The de-identified data will therefore be the underlying source
-for all cleaned and constructed data.
-It will also become the template dataset for final public release.
-
-%------------------------------------------------
+Secure storage of the raw\sidenote{\url{https://dimewiki.worldbank.org/wiki/Data\_Security}}
+data means access to it will be restricted even inside the research team.
+Loading encrypted data multiple times it can be annoying.
+To facilitate the handling of the data, remove any personally identifiable information from the data set.
+This will create a de-identified data set, that can be saved in a non-encrypted folder. 
+De-identification,\sidenote{\url{https://dimewiki.worldbank.org/wiki/De-identification}}
+at this stage, means stripping the data set of direct identifiers such as names, phone numbers, addresses, and geolocations.\sidenote{\url{ https://www.povertyactionlab.org/sites/default/files/resources/J-PAL-guide-to-deidentifying-data.pdf}}
+The resulting de-identified data will be the underlying source for all cleaned and constructed data.
+Because identifying information is typically only used during data collection, 
+to find and confirm the identity of interviewees, 
+de-identification should not affect the usability of the data.
+In fact, most identifying information can be converted into non-identified variables for analysis purposes
+(e.g. GPS coordinates can be translated into distances). 
+However, if sensitive information is strictly needed for analysis,
+all the tasks described in this chapter must be performed in a secure environment.
```

Comment:
What is a _secure environment_? I have an idea, but we haven't explained that concept enough in this book to be able to say that here and not explain it further. What exactly is it you refer to here?

Reply:
Hard to say -- this should be defined in IRB documentation and/or organizational guidance. In some cases it's sufficient to do on a company-managed laptop but sometimes you would need like a cold machine.

Reply:
added examples in d09c32d

Reply:
I was mostly just pointing out that we kind of introduced a new concept _secure environment_ without explaining it. I think we are fine here if we just keep using the same terminology we have used before.

"_However, if sensitive information is strictly needed for analysis, then we need to keep our data encrypted while doing the tasks described in this chapter._"

(somehow the suggestion function was not working for me)

Reply:
Changed in 7527ffa